main.go

```diff
@@ -239,7 +239,7 @@ func scan(cmd *cli.Cmd) {
 			f, err := os.Create(*outputFile)
 			defer f.Close()
 			if err != nil {
-				log.Fatalf("Could not open output file %s: %s", outputFile, err)
+				log.Fatalf("Could not open output file %s: %s", *outputFile, err)
```

Comment:
These changes were recommended by `go vet`.

Reply:
it was (unsurprisingly) correct.